packages/frontend/app/lib/session.server.ts

```diff
@@ -0,0 +1,17 @@
+import { createCookieSessionStorage } from '@remix-run/node'
+
+const SESSION_NAME = '__session'
+
+const { getSession, commitSession, destroySession } =
+  createCookieSessionStorage({
+    cookie: {
+      name: '__session',
+      httpOnly: true,
+      path: '/',
+      sameSite: 'lax',
+      secure: process.env.NODE_ENV === 'production',
```

Comment:
@mkurapov Is there anything to test here? I recall some shenanigans with the snackbar message cookie and some scenario. Mobile perhaps? Didn't notice any actual problems in testing.